ldi-orchestrator/ldio-common/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/PipelineConfig.java

```diff
@@ -5,9 +5,9 @@
 
 import java.util.LinkedList;
 import java.util.List;
-import java.util.Map;
 
 public class PipelineConfig {
```

Comment:
At this point, we may consider naming this "Pipeline" instead of "PipelineConfig"